docs/installation.md

```
@@ -546,7 +546,18 @@ Agent Zero needs to be configured further to redirect some functions to the Dock
 <img src="res/setup/image-23-1.png" alt="run ui" width="400"/>
 <br><br>
 
-      
+### Building a Native Installer
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)